src/System.Collections.Concurrent/src/System.Collections.Concurrent.csproj

```diff
@@ -15,27 +16,26 @@
     <Compile Include="System\Collections\Concurrent\CDSCollectionETWBCLProvider.cs" />
     <Compile Include="System\Collections\Concurrent\ConcurrentBag.cs" />
     <Compile Include="System\Collections\Concurrent\ConcurrentDictionary.cs" />
-    <Compile Include="System\Collections\Concurrent\ConcurrentQueue.cs" />
```

Comment:
You will need to keep these .cs files in for uapaot build until the change propagates to ProjectN.